Ryujinx.HLE/DeviceMemory.cs

```diff
@@ -98,6 +99,11 @@ public unsafe void WriteUInt64(long position, ulong value)
             *((ulong*)(_ramPtr + position)) = value;
         }
 
+        public unsafe void WriteStructure<T>(long position, T value)
```

Comment:
We name that WriteStruct/ReadStruct everywhere else so we must keep that consistent here

```suggestion
        public unsafe void WriteStruct<T>(long position, T value)
```